src/card.scss

```diff
@@ -0,0 +1,367 @@
+      margin-right: $fd-card-analytics-spacing;
+    }
+  }
+
+  &__analytics-text {
+    @include fd-reset();
+
+    padding-bottom: 0.25rem;
+    color: var(--sapTile_TextColor);
+  }
+
+  &__analytics-content {
+    @include fd-reset();
+
+    color: var(--sapTile_TitleTextColor);
+  }
+
+  &__second-subtitle {
+    @include fd-reset();
+    @include fd-ellipsis();
+
+    font-size: var(--sapFontSmallSize);
+    color: var(--sapTile_TextColor);
+    margin-top: 0.25rem;
+    width: 100%;
+  }
+
+  .#{$numeric-content} {
+    width: auto;
+  }
+
+  &__content {
+    @include fd-reset();
+
+    flex: 1 0 auto;
+  }
+
+  &__content-container {
+    @include fd-reset();
+    @include fd-flex(column);
+
+    &--horizontal {
+      flex-direction: row;
+
+      .#{$block}__content-section {
+        margin-bottom: 0;
+        margin-right: $fd-card-sections-horizontal-spacing;
+
+        &:last-child {
+          margin-right: 0;
+        }
+
+        @include fd-rtl() {
+          margin-right: 0;
+          margin-left: $fd-card-sections-horizontal-spacing;
+
+          &:first-child {
+            margin: 0;
+          }
+        }
+      }
+    }
+  }
+
+  &__content-section {
+    @include fd-reset();
+    @include fd-flex(column);
+
+    min-width: 12rem;
+    max-width: 24rem;
+    width: 100%;
+    margin-bottom: $fd-card-sections-vertical-spacing;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__content-title {
+    @include fd-reset();
+
+    padding-bottom: 0.5rem;
+  }
+
+  &__content-group {
+    @include fd-reset();
+
+    @include fd-flex() {
+      align-items: center;
+    }
+
+    margin-top: 1rem;
+  }
+
+  &__content-group-text {
+    @include fd-reset();
+    @include fd-flex(column);
+
+    margin-left: $fd-card-content-avatar-text-spacing;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    @include fd-rtl() {
+      margin-left: 0;
+      margin-right: $fd-card-content-avatar-text-spacing;
+
+      &:first-child {
+        margin: 0;
+      }
+    }
+  }
+
+  &__content-group-label {
+    @include fd-reset();
+  }
+
+  &__content-group-value {
+    @include fd-reset();
+
+    margin-top: 0.25rem;
+  }
+
+  &--analytical {
+    .#{$block}__subtitle-area {
+      margin-top: 0.5rem;
+    }
+
+    .#{$block}__title {
+      @include line-clamp(2);
+    }
+  }
+
+  &--table {
+    &.#{$block}--compact {
+      .#{$block}__content {
+        padding-bottom: 0.5rem;
+      }
+    }
+  }
+
+  &--object {
+    .#{$block}__content {
+      padding: 1rem;
+    }
+  }
+
+  // IE10+ fixes for line-clamp
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    .#{$block}__title {
+      max-height: 4rem;
+      flex: 1;
+    }
+
+    .#{$block}__subtitle {
+      max-height: 2.5rem;
+      flex: 1;
+    }
+
+    .#{$block}--analytical {
+      .#{$block}__title {
+        max-height: 3rem;
+      }
+    }
+
+    .#{$numeric-content} {
+      width: 100%;;
+    }
+  }
+}
```

Comment:
I know that it works well, but wasn't it forbidden to use ie11 specific media queries?

Reply:
line-clamp is not working in IE 11. Without this code the header title and subtitle just grow infinitely to fit the text.

Reply:
I removed the IE specific rules, added the max-height to the title and subtitle and some other tricks that will make it work in all browsers :)